assets/js/medialib_content.js

```diff
@@ -0,0 +1,7 @@
+const content = document.getElementsByClassName('category')
+for (let i = 0; i < content.length; i++) {
+  const header = content[i].getElementsByClassName('header')
+  header[0].addEventListener('click', function () {
+    content[i].classList.toggle('active')
+  })
+}
```

Comment:
Just as a note; For readability, you could have used forEach and querySelector; 
And also try to keep names the same to reduce complexity - `content` vs `category`

```
const categories = document.querySelectorAll('.category');

categories.forEach(category => {
  const header = category.querySelector('.header');
  header.addEventListener('click', () => {
    category.classList.toggle('active');
  });
});
```